src/demo/containers/playground/PeopleProfileDemo.tsx

```diff
@@ -13,6 +14,16 @@ export const PeopleProfileDemo = () => {
             summaryLink={"/Explore/People"}
           />
         </div>
+        <div className="row">
+          <UserCardListRotate
+            sql={"SELECT * FROM syn13897207 where isFeatured=true"}
+            count={3}
+            useQueryResultUserData={true}
+            size={MEDIUM_USER_CARD}
+            summaryLinkText={"EXPLORE ALL PEOPLE"}
+            summaryLink={"/Explore/People"}            
+          />
+        </div>
       </div>
```

Comment:
output:
<img width="1426" alt="Screen Shot 2020-09-17 at 3 57 04 PM" src="https://user-images.githubusercontent.com/1864447/93537025-49feb680-f8ff-11ea-84ea-b5efe2af3fc2.png">

Reply:
Looking at the small cards, it seems that the background is transparent so the pink color becomes the card's background. I think it should be white instead.

Reply:
Good catch, I'll take a look...

Reply:
Updated!  
<img width="1203" alt="Screen Shot 2020-09-17 at 4 37 48 PM" src="https://user-images.githubusercontent.com/1864447/93538951-4883bd00-f904-11ea-85a7-90729bcfd011.png">